Formula/hyx.rb

```diff
@@ -0,0 +1,22 @@
+class Hyx < Formula
+  desc "Powerful hex editor for the console"
+  homepage "https://yx7.cc/code/"
+  url "https://yx7.cc/code/hyx/hyx-2021.06.09.tar.xz"
+  sha256 "8d4f14e58584d6cc8f04e43ca38042eed218882a389249c20b086730256da5eb"
+  license "MIT"
+
+  def install
+    inreplace "Makefile", " -Wl,-s", " "
+    inreplace "Makefile", " -Wl,-z,relro,-z,now -fpic -pie", " "
+    inreplace "Makefile", " -Werror", " "
```

Comment:
These all need to be submitted upstream

Reply:
Could I just not use the Makefile at all? (Those options are GCC specific and aren't available or depreciated on the macOS linker.)

Reply:
It could literally be `cc *.c -o hyx` (well, the Homebrew equivalent to replace cc with $CC).

Reply:
Fixed in https://github.com/Homebrew/homebrew-core/pull/106085/commits/6405e291d009b3d735399ad629515439990d4fa4 with environment-based workaround.  Issue reported upstream also.